packages/v2/gui/src/fakta/sykdom-og-opplæring/2-sykdom/SykdomUperiodisertFerdigvisning.tsx

```diff
@@ -3,31 +3,52 @@ import { LabelledContent } from '../../../shared/labelled-content/LabelledConten
 import type { UperiodisertSykdom } from './SykdomUperiodisertForm';
 import { VurdertAv } from '../../../shared/vurdert-av/VurdertAv';
 import { ICD10 } from '@navikt/diagnosekoder';
-
+import { Lovreferanse } from '../../../shared/lovreferanse/Lovreferanse';
 const SykdomUperiodisertFerdigvisning = ({ vurdering }: { vurdering: UperiodisertSykdom }) => {
+  const sykdomGodkjentText = () => {
+    if (vurdering.godkjent === 'ja') {
+      return 'Ja';
+    }
+    if (vurdering.godkjent === 'nei') {
+      return 'Nei';
+    }
+    if (vurdering.godkjent === 'mangler_dokumentasjon') {
+      return 'Mangler dokumentasjon';
+    }
+    return '';
+  };
   return (
     <div className="flex flex-col gap-4">
-      <LabelledContent
-        label="Vurder om barnet har en funksjonshemning eller en langvarig sykdom antatt å vare i mer enn ett år som følge av § 9-14."
-        indentContent
-        size="small"
-        content={
-          <BodyLong size="small" className="whitespace-pre-wrap">
-            {vurdering.begrunnelse}
-          </BodyLong>
-        }
-      />
-      <VurdertAv ident={vurdering.vurdertAv} date={vurdering.vurdertTidspunkt} size="small" />
+      <div>
+        <LabelledContent
+          label={
+            <>
+              Vurder om barnet har en funksjonshemning eller en langvarig sykdom antatt å vare i mer enn ett år som
+              følge av <Lovreferanse>§ 9-14</Lovreferanse>.
```

Comment:
Denne setningen høres for meg ut som at § 9-14 er skyld i at barnet har funksjonshemming eller en langvarig sykdom.